instagram/src/components/PostContainer/posts.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PostHeader from './PostHeader';
+const Posts = props => {
+    return (
+        <div className="post-border">
+            <PostHeader
+             username={props.post.username}
+             thumbnailUrl={props.post.thumbnailUrl}
+            />
+        <div className="post-image-wrapper">
+            <img
+                alt="post thumbnail"
+                className="post-image"
+                src={props.post.imageUrl}
+            />
+        </div>
+        <CommentSection comments={props.post.comments} />
```

Comment:
You will have to import this. But right now importing it will break it because there's nothing inside the CommentSection file. If I comment out this line, everything else renders fine.